src/pyhf_combine_converter/pyhf_convert_to_datacard.py

```diff
@@ -10,7 +10,6 @@
 import json
 from optparse import OptionParser
 try:
-    from HiggsAnalysis.CombinedLimit.DatacardParser import *
```

Comment:
@peterridolfi can you confirm tomorrow that this works with the tests that you have so far?